### PR TITLE
added Eslint, Prettier and env config

### PR DESCRIPTION
**Pull request by Mauricio-Natacci**
Added Prettier config
Added extension recommendations
Added EsLint config
Added autocomplete environment variables

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "dev": "concurrently \"npm run server\" \"npm run client\"",
     "data:import": "node backend/seeder",
     "data:destroy": "node backend/seeder -d",
-    "heroku-postbuild": "NPM_CONFIG_PRODUCTION=false npm install --prefix frontend && npm run build --prefix frontend"
+    "heroku-postbuild": "NPM_CONFIG_PRODUCTION=false npm install --prefix frontend && npm run build --prefix frontend",
+    "prettier-format": "prettier --config .prettierrc '*/**/*.ts' --write"
```
Comment (nathan-torquato) [inline package.json]: don't we have a script for running eslint too?

File: to-do.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-1. colocar arquivo de configuracao de eslint no projeto para manter padrao
-1. colocar arquivo de configuracao de prettier no projeto para manter padrao
-1. colocar arquivo de configuracao de vs code no projeto para sugerir extensions e configuracao de format file on save
-1. fazer com que os nomes das variaveis de ambiente customizadas aparecam no autocomlete ao digitar "process.env" em arquivos `.ts`
+1. colocar arquivo de configuracao de eslint no projeto para manter padrao ✅
+1. colocar arquivo de configuracao de prettier no projeto para manter padrao ✅
+1. colocar arquivo de configuracao de vs code no projeto para sugerir extensions e configuracao de format file on save ✅
+1. fazer com que os nomes das variaveis de ambiente customizadas aparecam no autocomlete ao digitar "process.env" em arquivos `.ts` ✅
```
Comment (nathan-torquato) [inline to-do.md:4]: nice icons.. no related TODOs in the code, right?

Comment (Mauricio-Natacci) [reply to-do.md:4]: yeah, there are no TODOs related to this.

File: package.json (hunk 1 above)
Comment (Mauricio-Natacci) [reply package.json]: you're right! 
script added!

**Review by nathan-torquato — APPROVED** [review]

**Pull request merged**